pkgs/tools/inputmethods/interception-tools/default.nix

```
@@ -2,27 +2,25 @@
   libevdev, udev }:
 
 let
-  version = "0.2.1";
+  version = "0.6.6";
   baseName = "interception-tools";
 in stdenv.mkDerivation {
   name = "${baseName}-${version}";
```

```suggestion
stdenv.mkDerivation rec {
  pname = "interception-tools";
  version = "0.6.6";
```
Remove the `let` binding.

GitHub won't let me apply suggestions to deleted lines. -.-